synapse/handlers/room_list.py

```diff
@@ -87,14 +89,15 @@ def get_local_public_room_list(self, limit=None, since_token=None,
         return self.response_cache.wrap(
             key,
             self._get_public_room_list,
-            limit, since_token, network_tuple=network_tuple,
+            limit, since_token,
+            network_tuple=network_tuple, from_federation=from_federation,
         )
 
     @defer.inlineCallbacks
     def _get_public_room_list(self, limit=None, since_token=None,
                               search_filter=None,
                               network_tuple=EMPTY_THIRD_PARTY_ID,
-                              timeout=None,):
+                              from_federation=False, timeout=None,):
```

Comment:
```suggestion
                              from_federation=False, 
                              timeout=None,
    ):
```

(or whatever the right alignment for the closing paren is with this style)